clusters/hive/pools/rhtap/rhtap-ocp-4-14-0-amd64-aws-us-east-2_clusterpool.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  creationTimestamp: null
+  labels:
+    architecture: amd64
+    cloud: aws
+    owner: rhtap
+    product: ocp
+    region: us-east-2
+    version: "4.14"
+    version_lower: 4.14.0-0
+    version_upper: 4.15.0-0
+  name: rhtap-ocp-4-14-amd64-us-east-2
+  namespace: rhtap-cluster-pool
+spec:
+  baseDomain: ci.stonesoupengineering.com
+  hibernationConfig:
+    resumeTimeout: 20m0s
+  imageSetRef:
+    name: ocp-release-4.14.11-x86-64-for-4.14.0-0-to-4.15.0-0
+  installAttemptsLimit: 1
+  installConfigSecretTemplateRef:
+    name: rhtap-aws-us-east-2
+  labels:
+    tp.openshift.io/owner: rhtap
+  maxSize: 20
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: rhtap-aws-credentials
+      region: us-east-2
+  pullSecretRef:
+    name: pull-secret
+  runningCount: 1
+  size: 10
```

Comment:
IMO the size of a cluster pool **10** is too much for testing of a [single job](https://github.com/openshift/release/pull/48415/files#diff-142dad0885404df83d7c20686ac963d5b39a0a6e203fe6b0b338989d53506464R56). I'd start with 5. If it's too low, we can increase the size later. Wdyt?

Reply:
I changed it to 5.